src/main/java/se/sundsvall/oepintegrator/service/WebMessageService.java

```diff
@@ -0,0 +1,43 @@
+package se.sundsvall.oepintegrator.service;
+
+import static org.zalando.problem.Status.BAD_REQUEST;
+import static se.sundsvall.oepintegrator.utility.Constants.REFERENCE_FLOW_INSTANCE_ID;
+
+import callback.AddMessageResponse;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import org.zalando.problem.Problem;
+import se.sundsvall.oepintegrator.api.model.webmessage.ExternalReference;
+import se.sundsvall.oepintegrator.api.model.webmessage.WebMessageRequest;
+import se.sundsvall.oepintegrator.integration.db.model.enums.InstanceType;
+import se.sundsvall.oepintegrator.integration.opene.soap.OpeneSoapIntegration;
+import se.sundsvall.oepintegrator.service.mapper.MessageMapper;
+
+@Service
+public class WebMessageService {
+
+	private final OpeneSoapIntegration openeSoapIntegration;
+
+	public WebMessageService(final OpeneSoapIntegration openeSoapIntegration) {
+		this.openeSoapIntegration = openeSoapIntegration;
+	}
+
+	public AddMessageResponse createWebMessage(final String municipalityId, final InstanceType instanceType, final WebMessageRequest request, final List<MultipartFile> files) {
```

Comment:
Is it possible to we encapsulate the opene-stuff in this class? 
I don't think it's necessary to pullute the API (of this class) with openE-specific stuff?


I.e.:
Do we need to return AddMessageResponse from createWebMessage method? I noticed that the AddMessageResponse.ID is used in the WebMessageResource class. Could we just return this value instead?